challenges/towers_of_hanoi/README.md

```diff
@@ -0,0 +1,36 @@
+Read me
+# Towers of hanoi
+
+**Author** : Jay Adams
+**Version**: 0.1.0
+
+## Overview
+The Towers of Hanoi is a mathematical puzzle where you have 3 towers and N disks of different sizes.
+The puzzle starts with the disks sorted (i.e. smaller disks sits on top of larger disks) on the left-most tower, and the objective of the puzzle is to move the disks to the right-most tower with the following restrictions:
+Only one disk can be moved at a given time.
+A disk is moved from the top of a tower to the top of another tower.
+A disk can’t be placed on top of a smaller disk.
+
+
+
+## Getting Started
+Do all your work in a public repository called data-structures-and-algorithms, with a well-formated, detailed top level README.md matching the example provided by your instructor
+Create a branch in your repository called multi-bracket-validation
+On your branch, create…
+
+Python: a file called towers_of_hanoi.py
```

Comment:
This "getting started" is supposed to be how a _user_ should use your code, not how you got started.